src/widgets/DataTable/DataTable.js

```diff
@@ -69,11 +69,13 @@ const DataTable = React.memo(({ renderRow, renderHeader, style, data, columns, .
     return newRef;
   };
 
-  // Focuses the next editable cell in the list. Back to the top on last row.
+  // Focuses the next editable cell in the list. On the last row, dismiss the keyboard.
   const focusNextCell = refIndex => {
+    if (refIndex + 1 === numberOfEditableCells) return Keyboard.dismiss();
```

Comment:
I always get scared by `===` in these contexts because of possible off-by-one errors.

Maybe:

```
const maxIndex = numberOfEditableCells - 1;
if (refIndex >= maxIndex) return Keyboard.dismiss();
```

Reply:
0ae68dcedb4cbaafc17fe7ec415ad690259577ff

I've actually thought about this a bit - I prefer the off-by-one error actually get thrown rather than silently ignored, when values should be specific. Used variables though